modules/common/css/lib/forms.scss

```diff
@@ -82,6 +82,7 @@
       color: $wdc-color-black-pepper-300;
       font-family: 'Roboto', 'Helvetica Neue', 'Helvetica', Arial, sans-serif;
       position: relative;
+      top: 2px;
```

Comment:
What was this for? This is the only difference between the react and css versions now (visually at least)

Reply:
So it seems the react version changed and they're not correct. The CSS is the correct version and I've made this change to align the CSS to the React. I'm still investigating.